src/app.py

```diff
@@ -1,13 +1,22 @@
-import sys
-from PyQt5.QtWidgets import QApplication
-
-from widgets.main_layout import RandomGenerator
+from database.database_writer import Writer
+from database.database_reader import Reader
+from database.database_data_updater import DocumentUpdater
```

Comment:
I don't want to be niggling, but I think this solution is prettier. I think we should choose one convention.

```suggestion
from database import Writer, Reader, DocumentUpdater
```